README.md

```diff
@@ -188,6 +189,8 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Java Blog](http://binkurt.blogspot.com.tr/) - Binnur Kurt
 - [Java Blog](https://mericberber.com/javase/) - Meriç BERBER
 - [Java ve Java Teknolojileri ile alakalı çözümler](http://ofarukkurt.blogspot.com/) - Ömer Faruk Kurt
+- [Java, Java Kütüphaneleri ve Java Teknolojileri](http://ilkaygunel.com) - İlkay GÜNEL 
```

Comment:
"Java, Java Kütüphaneleri ve Java Teknolojileri" başlığı verilen siteyle çok da örtüşmüyor gibi. Üstteki gibi bir konu ya da bir etiket olsa olabilirdi de anasayfa sadece sizin kim olduğunuzu gösteriyor şu an için. Hiç olmazsa /blog olarak değiştirelim derim.